services/order.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
```

Comment:
The use of type hints like `list[dict]` requires Python 3.9 or later. Ensure your environment is using a compatible version to avoid runtime issues.